### PR TITLE
Fix item loading

### PR DESCRIPTION
**Pull request by bdlm-dev**
`Type.GetMethod` can't access private, static methods without the appropriate passed bitflags.

### CODE REVIEW

File: Winch/Winch.csproj [hunk 1]
```diff
@@ -17,8 +17,9 @@
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="DredgeGameLibs" Version="1.0.4.1" />
 		<PackageReference Include="Lib.Harmony" Version="2.2.2" />
-		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```
Comment (SloopyNoSleep) [inline Winch/Winch.csproj]: Any particular reason for kicking up the version in this pr?

Comment (SloopyNoSleep) [reply Winch/Winch.csproj]: I'm not sure I like how the phrasing of that comment sounds. Let me rephrase:
This increase in version seems accidental/incidental. Is there are reason for it for this pr? I don't think it needs to be removed, just curious

**Review by SloopyNoSleep — COMMENTED** [review]
I don't think there is any particular problem with the PR having this many commits since the overall delta is just a small set of changes. I'm just mentioning it just in case it may muddle anything. I believe Github (and git in general) is just going to squash this, right?

File: Winch/Winch.csproj (hunk 1 above)
Comment (Ashiepaws) [reply Winch/Winch.csproj]: I specifically chose that version (13.0.1) as it's the one used by DREDGE, didn't know if it made a huge difference but just wanted to make sure.

**Comment by Ashiepaws** [comment]
Yeah, they'll be squashed, no need to worry.

File: Winch/Winch.csproj (hunk 1 above)
Comment (SloopyNoSleep) [reply Winch/Winch.csproj]: Ah I see, thanks for clarifying!

Comment (bdlm-dev) [reply Winch/Winch.csproj]: Not any reasoning to it, feel free to change it back to 13.0.1

Comment (SloopyNoSleep) [inline Winch/Winch.csproj]:
```suggestion
		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment (SloopyNoSleep) [reply Winch/Winch.csproj]: Hmm dunno if this works like in azure where the PR author can just quickly accept suggestion and commit

Comment (bdlm-dev) [reply Winch/Winch.csproj]: 😁

Comment (bdlm-dev) [reply Winch/Winch.csproj]: Named that commit badly, my bad.

Comment (SloopyNoSleep) [reply Winch/Winch.csproj]: Time to rebase! 😈

Comment (bdlm-dev) [reply Winch/Winch.csproj]: I definitely, for sure, 100%, absolutely know what that is and how to do it, definitely.
